### DISCUSSION TITLE
<BambuHandy app webcam>

### DISCUSSION THREAD

**Discussion by AlexVikingos** [X1Plus/X1Plus#144] [Q&A]
X1Plus firmware web camera quality is poor and only works from orca and bambu slicer, but not from mobile app in iPhone

**Comment by dmiart1235123** [comment]
Mine is the same quality on Android as on orca/bambu. Maybe some sort of wifi connectivity problem on iphone's end?

**Comment by campmeharder** [comment]
I have the opposite, works well on handy app, but the bambustudio on linux is choppy and poor image.

**Comment by balika011** [comment]
i can't reproduce the issue

**Comment by jwise** [comment]
Converting to discussion; WiFi performance is likely improved in #84 and #85, but it's not clear that this is a problem with X1Plus wifi as much as it is just this printer's wifi being kind of flaky.  Can you reproduce this regression:

1) after you boot into stock FW?
2) after installing the brcmfmac test fw?
